ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,36 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.fft2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        shape=(2, 2),
```

Comment:
Input array is not limited to only `(2, 2)` isn't it? Please widen the test range :pray: